spec/README.md

```diff
@@ -186,6 +186,37 @@ Field Name | Type | Description
 <a name="parametersDefault"></a>default | `string` | **Optional** An optional default value for the parameter.
 <a name="parametersHidden"></a>hidden | `string` | **Optional** An optional boolean signifying the parameter should be obscured when displayed.
 
+##### XPathing
```

Comment:
"XPathing" doesn't seem to be a common, well-known term for this. How about "XPath Reference"?